<launch>
  <!-- 1) RealSense driver with depth‐to‐color alignment enabled -->
  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <arg name="align_depth" value="true"/>
  </include>

  
  <!-- Spustenie YOLO detekčného uzla -->
  <node pkg="yolo_tomato_detector" type="detect_yolo_stream.py" name="yolo_detector" output="screen">
    <param name="image_encoding" value="bgr8"/>
  </node>
</launch>
